cmd/privateendpoints-setuplink.go

```diff
@@ -33,8 +36,8 @@ var privateEndpointsSetupLinkCmd = &cobra.Command{
 		cloudCluster := cluster.(*clouddeploy.ClusterInfo)
 
 		if shouldAutoConfig {
-			if instanceId != "" || vmId != "" {
-				logger.Fatal("must not specify both auto and instance-id/vm-id")
+			if instanceId != "" || vmId != "" || instanceIdGcp != "" {
```

Comment:
What's the reason for adding an additional parameter here when GCP uses the same terminology as AWS?  You should be able to determine the 'meaning' in terms of AWS vs GCP trivially by looking at the clouddeploy.ClusterInfo to know what kind of cluster this is.  Ultimately the instance-id and vm-id are really just aliases of eachother for convenience.